market-contract/src/data_structures/order.sw

```diff
@@ -28,7 +28,6 @@ impl Order {
         order_type: OrderType,
         owner: Identity,
         price: u64,
-        price_decimals: u32,
```

Comment:
Removed `price_decimals` as an input to `Order::new()` to remove the limitation on the minimum price. Minimum price is 1e-9.